cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -564,21 +560,22 @@
         margin-top: @grid_gutter-width;
     }
 
-    .note-ctr {
+    .credit-note-ctr {
         border-top: 1px solid @gray-10;
     }
 
-    .note {
-        .grid_column( 8 );
-        margin: 0;
-        padding: 0 0 0 25px;
+    .credit-note {
```

Comment:
This really should be a modifier on `m-notification`. We shouldn't have components that completely transform appearance without a modifier. Maybe update `.credit-note` to `.m-notification__borderless` or something and add a comment above it that it's an enhancement of `m-notification`. Eventually we'd migrate that into CF if this a sanctioned pattern that might find use outside of BAH.

Reply:
Fixed Removed ids in 70e143a.